docs/source/conf.py

```diff
@@ -86,6 +86,8 @@
 # TODO: document these and remove them from here.
 
 coverage_ignore_functions = [
```

Comment:
could we make this list a tuple of module name and the ignored functions please?